node-test/src/test/scala/org/bitcoins/node/networking/P2PClientTest.scala

```diff
@@ -130,11 +123,19 @@ class P2PClientTest
   behavior of "P2PClient"
 
   override def beforeAll(): Unit = {
-    ChainDbManagement.createHeaderTable()
+    for {
+      _ <- ChainDbManagement.createHeaderTable()
+      _ <- ChainDbManagement.createFilterHeaderTable()
+      _ <- ChainDbManagement.createFilterTable()
+    } yield ()
   }
 
   override def afterAll(): Unit = {
-    ChainDbManagement.dropHeaderTable()
+    for {
+      _ <- ChainDbManagement.dropFilterTable()
+      _ <- ChainDbManagement.dropFilterHeaderTable()
+      _ <- ChainDbManagement.dropHeaderTable()
```

Comment:
Might be nice to give `ChainDbManagement` some util methods to do this (and the above)